homeassistant/components/icloud/__init__.py

```diff
@@ -74,9 +72,6 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Optional(CONF_WITH_FAMILY, default=DEFAULT_WITH_FAMILY): cv.boolean,
         vol.Optional(CONF_MAX_INTERVAL, default=DEFAULT_MAX_INTERVAL): cv.positive_int,
-        vol.Optional(
-            CONF_GPS_ACCURACY_THRESHOLD, default=DEFAULT_GPS_ACCURACY_THRESHOLD
-        ): cv.positive_int,
```

Comment:
We should mark this deprecated instead of removing it, not to break existing configurations.